src/ios/OpenIDAuth.m

```diff
@@ -91,7 +91,11 @@ - (nullable OIDAuthState *)getAuthState {
 
 -(void)handleNotification:(NSNotification *)notification
 {
-    [self logMessage:@"OpenIDAuth Error: handleNotification should not be called!"];
+    // For compatibility with iOS 10 and earlier
+    NSURL* url = [notification object];
+    if ([url.scheme isEqualToString:@"emission.auth"]) {
+        [self.currentAuthorizationFlow resumeAuthorizationFlowWithURL:url];
```

Comment:
don't you have to set `currentAuthorizationFlow` to `nil` if successful?
https://github.com/openid/AppAuth-iOS/blob/619bb7c7d5f83cc2ed19380d425ca8afa279644c/README.md#authorizing-ios